libs/langchain/langchain/schema/runnable.py

```diff
@@ -776,8 +776,17 @@ class RouterRunnable(
 ):
     runnables: Mapping[str, Runnable[Input, Output]]
 
-    def __init__(self, runnables: Mapping[str, Runnable[Input, Output]]) -> None:
-        super().__init__(runnables=runnables)
+    def __init__(
+        self,
+        runnables: Mapping[
+            str, Union[Runnable[Input, Output], Callable[[Input], Output]]
+        ],
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(
+            runnables={key: _coerce_to_runnable(r) for key, r in runnables.items()},
+            **kwargs,
```

Comment:
Are the kwargs being passed here for serialization purposes?

Reply:
Oops good catch